content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -37,10 +38,10 @@ const NewTabPage = React.createClass({
     document.title = "New Tab";
     setFavicon("newtab-icon.svg");
   },
-  componentWillReceiveProps(nextProps) {
-    if (nextProps.isReady && !this.state.renderedOnce) {
+  componentDidUpdate(nextProps) {
+    if (!renderedOnce && nextProps.isReady) {
```

Comment:
The problem was that componentDidUpdate doesn't take any params at here.  Changing 

```
nextProps.isReady
```

to 

```
this.props.isReady
```

seems to fix things.

That said, it's still not clear what is causing this.props.isReady, which makes this code a bit hard to reason about.  Can you add a comment describing why this is the right test?